jcl/src/ibm.healthcenter/share/classes/module-info.java

```diff
@@ -0,0 +1,46 @@
+/*[INCLUDE-IF JAVA_SPEC_VERSION > 8]*/
+/*******************************************************************************
+ * Copyright (c) 2021, 2021 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+/**
+ * Provides the agent that supports IBM Monitoring and Diagnostic Tools - Health Center.
+ */
+module ibm.healthcenter {
```

Comment:
Does this need to be added to the OpenJ9 project? Seems it should be added to the extensions.

Reply:
I had it in the extension repo originally, but I thought that one copy was better than many.

Reply:
It would be, except ibm.healthcenter isn't open source so the module definition doesn't belong in the Openj9 open source.

Reply:
Fixed in 5cd45ce0fea3.